src/main/js/wiki/components/PageHeader.tsx

```diff
@@ -57,16 +48,19 @@ class PageHeader extends React.Component<Props, State> {
     };
```

Comment:
showMoveForm and showDeleteConfirm are missing here

Reply:
- [x] done